apps/agent/src/main/screenshot.ts

```
@@ -0,0 +1,110 @@
+
+		const sourcesFull = await electron.desktopCapturer.getSources({
+			types: ['screen'],
+			thumbnailSize: args.screenSize
+		})
+
+		const screens = [];
+
+		sources.forEach((source) => {
+			if (
+				monitor &&
+				monitor.captured &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)